### PR TITLE
Lineage 15.1

### PR DESCRIPTION
**Pull request by shakhawat3742**
Latest

### CODE REVIEW

**Comment by shakhawat3742** [comment]
Best od

File: core/java/android/content/pm/PackageInfo.java [hunk 1]
```diff
@@ -286,8 +286,26 @@
...
+    public static final int FLAG_OVERLAY_STATIC = 1 << 1;
+
+    /**
+     * Flag for use with {@link #overlayFlags}. Marks the overlay as trusted (not 3rd party).
+     * @hide
+     */
+    public static final int FLAG_OVERLAY_TRUSTED = 1 << 2;
+
+    /**
+     * Modifiers that affect the state of this overlay. See {@link #FLAG_OVERLAY_STATIC},
```
Comment (shakhawat3742) [inline core/java/android/content/pm/PackageInfo.java]: Yes

**Comment by shakhawat3742** [comment]
Yes

**Comment by ItzLevvie** [comment]
@PerfidusPistor Close?

Please submit patches through LineageOS Gerrit

**Comment by ItzLevvie** [comment]
@shakhawat3742 Close?

Please submit patches through LineageOS Gerrit

**Comment by ItzLevvie** [comment]
Hey there, @shakhawat3742!

This is a reply, to let you know that LineageOS doesn't merge pull
requests submitted directly to their github.com repositories.

You can submit your change requests through their public review system,
located at https://review.lineageos.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, I also suggest you take
a look at https://wiki.lineageos.org/usinggerrit-howto.html

Thanks!

File: core/java/android/text/Layout.java [hunk 2]
```diff
@@ -1070,6 +1070,9 @@ private boolean primaryIsTrailingPrevious(int offset) {
             if (limit > lineEnd) {
                 limit = lineEnd;
             }
+            if (limit == start) {
+                continue;
```
Comment (tesoy-xx) [inline core/java/android/text/Layout.java]: https://github.com/topjohnwu/Magisk.git

**Comment by haggertk** [comment]
Code changes to LineageOS are submitted through Gerrit - https://wiki.lineageos.org/how-to/submitting-patches

**Pull request closed without merging**